src/base/cubrid_getopt.h

```diff
@@ -27,9 +27,14 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef ___GETOPT_H___
-#define ___GETOPT_H___
+#ifndef _CUBRID_GETOPT_H_
+#define _CUBRID_GETOPT_H_
 
+#include "config.h"
+
+#ifdef HAVE_GETOPT_H
+#include <getopt.h>
```

Comment:
according to the internet documentation I found (https://en.wikipedia.org/wiki/Getopt, http://pubs.opengroup.org/onlinepubs/9699919799/)
system <getopt.h> has declaration only for getopt_long()
getopt() is in <unistd.h>
I think you need to include it also